src/main/java/edu/cornell/kfs/gl/CuGeneralLedgerConstants.java

```diff
@@ -5,6 +5,7 @@
 public class CuGeneralLedgerConstants extends GeneralLedgerConstants {
 
 	public static final String ANNUAL_CLOSING_CHARTS_PARAM = "ANNUAL_CLOSING_CHARTS";
+	public static final String COLLECTOR_HEADER_RECORD_TYPE = "HD";
```

Comment:
It feels like GL Constants is a better place. I don't have a terribly strong opinion on this matter